tracer/src/Datadog.Trace/Agent/AgentWriter.cs

```diff
@@ -479,6 +486,11 @@ SpanBuffer SwapBuffers()
             }
 
             // All the buffers are full :( drop the trace
+            DropTrace(spans);
+        }
+
+        private void DropTrace(ArraySegment<Span> spans)
+        {
```

Comment:
I think we need a debug lot here as well as nobody has healt metrics enabled, WDYT?

Reply:
Perhaps this should be a warning log too with some diagnostic information, like the trace ID and the number of spans. The trace ID could help if there's a distributed trace and the spans from a remote service got through, but not this one

Reply:
I agree, probably want a debug log at the very least. Maybe warning, except that this _may_ occur in high-load scenarios, so we may not want to do the extra log in that scenario?